atlp-rwanda/atlp-devpulse-fn
# #01 Chore: Create empty React app with react router

### What does this PR does?

1. This PR indicates project folder structure and folders/files naming conversion.
## What to expect?

- All required folders were added. (you can change or add file names in the folders according to the task you have been assigned but keep naming convesion)
## How to test this PR?

1. Clone this [repo](https://github.com/atlp-rwanda/atlp-devpulse-fn.git)
2.  run the command git checkout Project-setup (after being merged you will no longer need this command)
3. run the command npm install
4. run the command npm run dev to start the server
